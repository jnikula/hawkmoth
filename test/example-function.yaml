directive-arguments:
- example-function.c
example-title: Function
example-priority: 40
