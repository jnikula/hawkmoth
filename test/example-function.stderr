WARNING: example-function.c:9: declaration of 'struct list' will not be visible outside of this function
WARNING: example-function.c:9: declaration of 'enum mode' will not be visible outside of this function
